core/types/transaction.go

```diff
@@ -266,9 +266,9 @@ func (s Transactions) GetRlp(i int) []byte {
 	return enc
 }
 
-// TxDifference returns a new set t which is the difference between a to b.
-func TxDifference(a, b Transactions) (keep Transactions) {
-	keep = make(Transactions, 0, len(a))
+// TxDifference returns a new set which is the difference between a to b.
```

Comment:
Thank you. Would be even better to change it to "...difference between a and b."

Reply:
Yup, done.